README.md

```diff
@@ -16,7 +16,7 @@ This monorepo workspace is managed using [Nx](https://nx.dev). See the `README`
 
 ### Requirements
 
-You'll need Node 14 and . It is recommended that you install node using [nvm](https://github.com/nvm-sh/nvm)).
+You'll need Node v14.17.6. It is recommended that you install node using [nvm](https://github.com/nvm-sh/nvm)).
```

Comment:
Is it really necessary to be this specific? `Node 14` should be good. We may want to bump to [node 16 ](https://endoflife.date/nodejs) at some point.

Reply:
Ok, up to you.  I thought we had to all work on the same exact version, but if 14's close enough, all good.